src/jest.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
       extends: ['plugin:jest/recommended'],
       rules: {
         // Define custom parcelLab rules
+        'jest/expect-expect': 'off',
```

Comment:
Sadly this will disable a very useful rule for non-react codebases that will tell us if a test is lacking the expect statement.

We should figure out a way to disable such rule only on react. I would move it to the react file rather than here.

Reply:
I'll rollback this change for now and we can further discuss if we really need this disabled.
There is always the alternative of using expect with react-testing-library even though it is not necessary, or we could simply add an exception to the project's eslint config file, when needed, as I can imagine that not all projects will be using this testing library.

Reply:
"jest/expect-expect" rule allows the customization of assertion function names, so that if an assertion function is not called expect we can add this function name here
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/expect-expect.md#assertfunctionnames
Do you have an example of a test where a `expect` is not expected?

Reply:
https://testing-library.com/docs/queries/about#types-of-queries

I'd say every `getBy...` and `findBy...` type of queries won't need if you want to make sure only one element is found in the DOM.
Cuz if more than one, or none are to be found, it will throw.

A simple example would be this one https://github.com/parcelLab/parcellab-trackandtrace-plugin/blob/main/src/components/general/loading/loading.test.tsx#L5

Reply:
Anyways, the test could be re-written to have expect

```
 expect(screen.getAllByTestId("loader")).toHaveLength(1);
 ```

Reply:
Nice! Thanks for clarifying Lucas!
The rewritten version with expect is more explicit about what is expected, making the test code easier to read.
I vote to keep this rule on.